Not directly related to this but I'm getting the following error when loading moveit:

```
[ERROR] [1433504838.775942782]: Link 'rh_mfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776000388]: Link 'rh_ffmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776027476]: Link 'rh_ffmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776063101]: Link 'rh_ffdistal' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776098811]: Link 'rh_rfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776129159]: Link 'rh_mfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776157383]: Link 'rh_mfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776190388]: Link 'rh_mfdistal' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776223425]: Link 'rh_lfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776253053]: Link 'rh_rfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776282747]: Link 'rh_rfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776315468]: Link 'rh_rfdistal' is not known to URDF. Cannot disable collisons.
```

--- a/sr_multi_moveit/left_sr_ur10_moveit_config/config/ompl_planning.yaml
+++ b/sr_multi_moveit/left_sr_ur10_moveit_config/config/ompl_planning.yaml
@@ -6,17 +6,17 @@ planner_configs:
     type: geometric::EST
     range: 0.0  # Max motion added to tree. ==> maxDistance_ default: 0.0, if 0.0 setup()
     goal_bias: 0.05  # When close to goal select goal, with this probability. default: 0.05
-#  LBKPIECEkConfigDefault:
-#    type: geometric::LBKPIECE
-#    range: 0.0  # Max motion added to tree. ==> maxDistance_ default: 0.0, if 0.0, set on setup()
-#    border_fraction: 0.9  # Fraction of time focused on boarder default: 0.9
-#    min_valid_path_fraction: 0.5  # Accept partially valid moves above fraction. default: 0.5
-#  BKPIECEkConfigDefault:
-#    type: geometric::BKPIECE
-#    range: 0.0  # Max motion added to tree. ==> maxDistance_ default: 0.0, if 0.0, set on setup()
-#    border_fraction: 0.9  # Fraction of time focused on boarder default: 0.9
-#    failed_expansion_score_factor: 0.5  # When extending motion fails, scale score by factor. default: 0.5
-#    min_valid_path_fraction: 0.5  # Accept partially valid moves above fraction. default: 0.5
+  LBKPIECEkConfigDefault:
+    type: geometric::LBKPIECE
+    range: 0.0  # Max motion added to tree. ==> maxDistance_ default: 0.0, if 0.0, set on setup()
+    border_fraction: 0.9  # Fraction of time focused on boarder default: 0.9
+    min_valid_path_fraction: 0.5  # Accept partially valid moves above fraction. default: 0.5
+  BKPIECEkConfigDefault:
+    type: geometric::BKPIECE
+    range: 0.0  # Max motion added to tree. ==> maxDistance_ default: 0.0, if 0.0, set on setup()
+    border_fraction: 0.9  # Fraction of time focused on boarder default: 0.9
+    failed_expansion_score_factor: 0.5  # When extending motion fails, scale score by factor. default: 0.5
+    min_valid_path_fraction: 0.5  # Accept partially valid moves above fraction. default: 0.5
   KPIECEkConfigDefault:
     type: geometric::KPIECE
     range: 0.0  # Max motion added to tree. ==> maxDistance_ default: 0.0, if 0.0, set on setup()
@@ -56,8 +56,8 @@ left_arm:
   planner_configs:
     - SBLkConfigDefault
     - ESTkConfigDefault
-   # - LBKPIECEkConfigDefault
-   # - BKPIECEkConfigDefault
+    - LBKPIECEkConfigDefault
+    - BKPIECEkConfigDefault
     - KPIECEkConfigDefault
     - RRTkConfigDefault
     - RRTConnectkConfigDefault
@@ -71,8 +71,8 @@ left_hand:
   planner_configs:
     - SBLkConfigDefault
     - ESTkConfigDefault
-   # - LBKPIECEkConfigDefault
-   # - BKPIECEkConfigDefault
+    - LBKPIECEkConfigDefault
+    - BKPIECEkConfigDefault
     - KPIECEkConfigDefault
     - RRTkConfigDefault
     - RRTConnectkConfigDefault
@@ -86,8 +86,8 @@ lh_mf:
   planner_configs:
     - SBLkConfigDefault
     - ESTkConfigDefault
-   # - LBKPIECEkConfigDefault
-   # - BKPIECEkConfigDefault
+    - LBKPIECEkConfigDefault
+    - BKPIECEkConfigDefault
     - KPIECEkConfigDefault
     - RRTkConfigDefault
     - RRTConnectkConfigDefault

--- a/sr_multi_moveit/left_sr_ur10_moveit_config/launch/demo.launch
+++ b/sr_multi_moveit/left_sr_ur10_moveit_config/launch/demo.launch
@@ -3,7 +3,7 @@
   <arg name="db" default="false" />
   <!-- By default, we are not in debug mode -->
   <arg name="debug" default="false" />
-  <arg name="limited" default="false" />
+  <arg name="limited" default="true" />
   <!-- Load the URDF, SRDF and other .yaml configuration files on the param server -->
   <include file="$(find left_sr_ur10_moveit_config)/launch/planning_context.launch">
 

--- a/sr_multi_moveit/left_sr_ur10_moveit_config/launch/move_group.launch
+++ b/sr_multi_moveit/left_sr_ur10_moveit_config/launch/move_group.launch
@@ -1,6 +1,6 @@
 <launch>
   <arg name="pipeline" default="ompl" />
-  <arg name="limited" default="false" />
+  <arg name="limited" default="true" />
   <include file="$(find left_sr_ur10_moveit_config)/launch/planning_context.launch">
 
     <arg name="limited" value="$(arg limited)" />

--- a/sr_multi_moveit/left_sr_ur10_moveit_config/launch/moveit_planning_and_execution.launch
+++ b/sr_multi_moveit/left_sr_ur10_moveit_config/launch/moveit_planning_and_execution.launch
@@ -2,7 +2,7 @@
   <arg name="planning" default="ompl" />
   <!-- By default, we are not in debug mode -->
   <arg name="debug" default="false" />
-  <arg name="limited" default="false" />
+  <arg name="limited" default="true" />
   <arg name="load_robot_description" default="true" />
   <include file="$(find left_sr_ur10_moveit_config)/launch/planning_context.launch">
 

--- a/sr_multi_moveit/left_sr_ur10_moveit_config/launch/planning_context.launch
+++ b/sr_multi_moveit/left_sr_ur10_moveit_config/launch/planning_context.launch
@@ -1,7 +1,7 @@
 <launch>
   <!-- By default we do not overwrite the URDF. Change the following to true to change the default behavior -->
   <arg name="load_robot_description" default="false" />
-  <arg name="limited" default="false" />
+  <arg name="limited" default="true" />
   <!-- The name of the parameter under which the URDF is loaded -->
   <arg name="robot_description" default="robot_description" />
   <!-- Load universal robot description format (URDF) -->

--- a/sr_multi_moveit/right_sr_ur10_moveit_config/launch/demo.launch
+++ b/sr_multi_moveit/right_sr_ur10_moveit_config/launch/demo.launch
@@ -3,7 +3,7 @@
   <arg name="db" default="false" />
   <!-- By default, we are not in debug mode -->
   <arg name="debug" default="false" />
-  <arg name="limited" default="false" />
+  <arg name="limited" default="true" />
   <!-- Load the URDF, SRDF and other .yaml configuration files on the param server -->
   <include file="$(find right_sr_ur10_moveit_config)/launch/planning_context.launch">
 

--- a/sr_multi_moveit/right_sr_ur10_moveit_config/launch/move_group.launch
+++ b/sr_multi_moveit/right_sr_ur10_moveit_config/launch/move_group.launch
@@ -1,6 +1,6 @@
 <launch>
   <arg name="pipeline" default="ompl" />
-  <arg name="limited" default="false" />
+  <arg name="limited" default="true" />
   <include file="$(find right_sr_ur10_moveit_config)/launch/planning_context.launch">
 
     <arg name="limited" value="$(arg limited)" />

--- a/sr_multi_moveit/right_sr_ur10_moveit_config/launch/moveit_planning_and_execution.launch
+++ b/sr_multi_moveit/right_sr_ur10_moveit_config/launch/moveit_planning_and_execution.launch
@@ -2,7 +2,7 @@
   <arg name="planning" default="ompl" />
   <!-- By default, we are not in debug mode -->
   <arg name="debug" default="false" />
-  <arg name="limited" default="false" />
+  <arg name="limited" default="true" />
   <arg name="load_robot_description" default="true" />
   <include file="$(find left_sr_ur10_moveit_config)/launch/planning_context.launch">
 

--- a/sr_multi_moveit/right_sr_ur10_moveit_config/launch/planning_context.launch
+++ b/sr_multi_moveit/right_sr_ur10_moveit_config/launch/planning_context.launch
@@ -1,7 +1,7 @@
 <launch>
   <!-- By default we do not overwrite the URDF. Change the following to true to change the default behavior -->
   <arg name="load_robot_description" default="false" />
-  <arg name="limited" default="false" />
+  <arg name="limited" default="true" />
   <!-- The name of the parameter under which the URDF is loaded -->
   <arg name="robot_description" default="robot_description" />
   <!-- Load universal robot description format (URDF) -->

